example/feedback/feedback.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Bucket feedback collection</title>
+    <script src="https://cdn.jsdelivr.net/npm/@bucketco/tracking-sdk@1"></script>
+    <style>
+      fieldset label {
+        display: flex;
+        align-items: center;
+        padding: 5px;
+        gap: 5px;
+      }
+
+      label input {
+        margin: 0;
+      }
+    </style>
+  </head>
+  <body>
+    <script>
+      function handleSubmit(e) {
+        e.preventDefault();
+
+        const formData = Object.fromEntries(new FormData(e.target).entries());
+
+        const feedbackPayload = {
+          featureId: "EXAMPLE_FEATURE",
+          userId: "EXAMPLE_USER",
+          companyId: "EXAMPLE_COMPANY",
+          score: formData.score ? Number(formData.score) : null,
+          comment: formData.comment ? formData.comment : null,
+        };
+
+        // Using the Bucket SDK
+        bucket.feedback(feedbackPayload);
+
+        // Using the Bucket API
```

Comment:
People have a tendency to blindly copy/paste things, so i suggest we comment out the full fetch example:

```
// Or using the Bucket HTTP API
/*
     fetch(....
     
     
*/

Reply:
Good point. will comment out